src/sentry/integrations/msteams/card_builder/base.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+from abc import ABC
+from typing import Any, Sequence
+
+from sentry.integrations.msteams.card_builder import Action, AdaptiveCard, Block
+from sentry.integrations.notifications import AbstractMessageBuilder
+
+from .block import create_text_block
+
+
+class MSTeamsMessageBuilder(AbstractMessageBuilder, ABC):  # type: ignore
+    def build(self) -> AdaptiveCard:
+        """Abstract `build` method that all inheritors must implement."""
+        raise NotImplementedError
+
+    def build_card(
```

Comment:
In the original design build was supposed to be inherited by all the different types of subclasses, I guess since we don't have that now, I can get rid of it and just rename `build_card` -> `build`.

Reply:
Updated.